### ISSUE TITLE
linux desktop launch problems since 2.16.0-1457

### ISSUE THREAD

**Issue by atemp** [RocketChat/Rocket.Chat.Electron#1352]
## My Setup
- Operating System: CentOS 7
- App Version: rocketchat-2.16.0-1457.x86_64
- Installation type: RPM

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description

After RPM update to 2.16.0-1457 one can not launch it anymore unless fixing file permissions manually.

### Current Behavior

```
> rocketchat-desktop 
[21245:1016/065457.499827:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/Rocket.Chat/chrome-sandbox is owned by root and has mode 4755.
```

If i fix the permissions manually it works. 

The problem exists on multiple different hosts, all running CentOS 7.

### Expected Behavior

It should start.

**Comment by orangerkater** [comment]
This happens at my set-up as well. I also run CentOS 7.

**Comment by sigsergv** [comment]
Try this command:

```
sudo chmod 4755 /opt/Rocket.Chat/chrome-sandbox
```

**Comment by sigsergv** [comment]
The same for dpkg package, command `sudo chmod 4755 /opt/Rocket.Chat/chrome-sandbox` fixes it, could be used as a workaround.

**Comment by atemp** [comment]
I am using the same workaround since the new version was released, but the drawback is: You have to apply it again and again after every update until the problem is fixed.

**Comment by franckadil** [comment]
I can confirm this issue.

**Comment by atemp** [comment]
problem was not fixed in `rocketchat-2.16.1-1470.x86_64`, update requires manual intervention on all hosts

**Comment by atemp** [comment]
since i fail to find the spec file used to create rocketchat RPMs, i created a workaround package that fixes permissions on updates:

```
%define name rocketchar-permission-fixer
%define version 1
%define release 1

Summary: Fixes broken rocketchat packages
Name: %{name}
Version: %{version}
Release: %{release}
License: public-domain
Group: System/Tools
Prefix: %{_prefix}
BuildArchitectures: noarch

%description
Contains hooks to fix broken rocketchat package that complain about
wrong sanbox permissions instead of starting:

> The SUID sandbox helper binary was found, but is not configured
> correctly. Rather than run without sandboxing I\'m aborting now. You
> need to make sure that /opt/Rocket.Chat/chrome-sandbox is owned by
> root and has mode 4755.

Bug is reported here:
https://github.com/RocketChat/Rocket.Chat.Electron/issues/1352

%post
if [ -f /opt/Rocket.Chat/chrome-sandbox ]
then
    chmod 4755 /opt/Rocket.Chat/chrome-sandbox
fi

%triggerin -- rocketchat
if [ -f /opt/Rocket.Chat/chrome-sandbox ]
then
    chmod 4755 /opt/Rocket.Chat/chrome-sandbox
else
    echo "%{name}-%{version}: TRIGGERIN: file not found: opt/Rocket.Chat/chrome-sandbox"
fi

%files
```

**Comment by atemp** [comment]
seems to be fixed in at  least 2.17.1-1548

**Comment by sigsergv** [comment]
It's not fixed to me, menu steals focus on Alt+Space shortcut.
